pipelines/scheduled/coverage/coverage.yml

```
@@ -5,7 +5,7 @@ steps:
         key:   "coverage"
         # Comment out the following `if:` statement for testing.
         # Please remember to uncomment it before merging the PR.
-        if: pipeline.slug == "julia-master-scheduled"
+        # if: pipeline.slug == "julia-master-scheduled"
```

Commit this change before merging the PR.
```suggestion
        if: pipeline.slug == "julia-master-scheduled"
```